src/android/java/io/jxcore/node/BtConnectorHelper.java

```diff
@@ -232,18 +241,21 @@ public void BeginConnectPeer(String toPeerId, ConnectStatusCallback connectStatu
         }
     }
 
+    //this is always called in context of thread that created instance of the library
```

Comment:
@DrJukka On line 248 what happens if bluetoothSocket is null? Currently the code does nothing. That is fine but in that case please rewrite the code to check for null and return and then out indent the rest of the code. I'll stop making this comment about 'if' styling (e.g. don't indent things like crazy, instead use return) and assume you will check all your files for it and please fix.